samcli/lib/build/workflows.py

```diff
@@ -4,7 +4,7 @@
 from typing import List
 
 CONFIG = namedtuple(
-    "Capability",
+    "CONFIG",
```

Comment:
is this right?

Reply:
I could ignore them but mypy complains. This is the "name-match" error mypy throws.